api/app/signals/settings/base.py

```diff
@@ -413,6 +416,11 @@
     },
 }
 
+CONSTANCE_CONFIG = {
```

Comment:
Do we want a Django Admin to edit these type of settings?

And also this could be used to make other settings configurable, think about the feature flags, etc. This way we can enable and disabled features without deployments. That would be super nice to have. 

One thing to take in consideration is that people have acces to the Django Admin.